buildspec.json

```diff
@@ -1,42 +1,48 @@
 {
     "dependencies": {
         "obs-studio": {
-            "version": "31.0.0-cmake-fix",
-            "baseUrl": "https://github.com/glikely/obs-studio/archive/refs/tags",
+            "version": "31.0.3-arm64-test",
+            "baseUrl": "https://github.com/anthony-linaro/obs-studio/archive/refs/tags",
             "label": "OBS sources",
```

Comment:
I've pushed the 31.0.3-arm64-test tag to my glikely/obs-studio fork. You can use that instead of having to carry it in your tree.